supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -16,19 +15,34 @@ import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.{Failure, Success, Try}
 
-case class SharedFileSystemRunState(status: String, date: Calendar) {
+sealed trait SharedFileSystemRunState {
+  def status: String
+  def terminal: Boolean = false
```

Comment:
[ToL] if I were writing this, I would not use a default so that someone extending this trait has to explicitly consider whether a newly-added state is terminal or not